src/taipy/core/_scheduler/_scheduler.py

```diff
@@ -63,8 +63,10 @@ def submit(
              callbacks: The optional list of functions that should be executed on jobs status change.
              force (bool) : Enforce execution of the pipeline's tasks even if their output data
                 nodes are cached.
-             wait (bool): Wait for the scheduled jobs created from the pipeline submission to be finished in asynchronous mode.
-             timeout (Union[float, int]): The optional maximum number of seconds to wait for the jobs to be finished before returning.
+             wait (bool): Wait for the scheduled jobs created from the pipeline submission to be finished in
+             asynchronous mode.
+             timeout (Union[float, int]): The optional maximum number of seconds to wait for the jobs to be finished
+             before returning.
```

Comment:
mkdocs expects a new line to have one more tab indentation so it can interpret it as the following of the previous line. 
Was it automatic formatting? If so, this is a problem we need to fix.
```suggestion
             wait (bool): Wait for the scheduled jobs created from the pipeline submission to be finished in
                 asynchronous mode.
             timeout (Union[float, int]): The optional maximum number of seconds to wait for the jobs to be finished
                 before returning.
```